src/Revamp.re

```diff
@@ -131,8 +129,7 @@ module Compiled = {
   let replaceByString = (re, replacement, input) =>
     Js.String.replaceByRe(re, replacement, input);
 
-  let split = (re, input) =>
-    Js.String.splitByRe(re, input);
+  let split = (re, input) => _splitByRe(re, input);
```

Comment:
This is what I meant:

```suggestion
  let split = (re, input) => Js.String.splitByRe(re, input)->Belt.Array.keepMap(x => x);
```

Might look odd, but if you look at signature of `leepMap` the function is expected to return an option, where `Some(x)` means the item is kept and `None` means it's dropped. Which makes it really easy for us to filter out the `None`s and unwrapping the `option`s in one go.